.goreleaser.yaml

```diff
@@ -3,42 +3,127 @@ before:
     - apt install musl-dev=1.2.2-1
+env:
+  - CGO_ENABLED=1
 builds:
-  - id: linux-amd64
+  - id: archwayd-linux-amd64
     main: ./cmd/archwayd
     binary: archwayd
     goos:
       - linux
     goarch:
       - amd64
     env:
-      - CGO_ENABLED=1
       - CC=x86_64-linux-gnu-gcc
+      - CXX=x86_64-linux-gnu-g++
     flags:
       - -mod=readonly
+      - -trimpath
     ldflags:
       - -s -w -X main.commit={{.Commit}} -X main.date={{ .CommitDate }} -X github.com/cosmos/cosmos-sdk/version.Name=archway -X github.com/cosmos/cosmos-sdk/version.AppName=archway -X github.com/cosmos/cosmos-sdk/version.Version={{ .Version }} -X github.com/cosmos/cosmos-sdk/version.Commit={{ .Commit }}
       - -linkmode=external
-      - -extldflags '-Wl,-z,muldefs -static -lm'
+      - -extldflags '-Wl,-z,muldefs --static -lm'
     tags:
       - netgo
       - muslc
-  - id: linux-arm64
+  - id: archwayd-linux-arm64
     main: ./cmd/archwayd
     binary: archwayd
     goos:
       - linux
     goarch:
       - arm64
     env:
-      - CGO_ENABLED=1
       - CC=aarch64-linux-gnu-gcc
+      - CXX=aarch64-linux-gnu-g++
     flags:
       - -mod=readonly
+      - -trimpath
     ldflags:
       - -s -w -X main.commit={{.Commit}} -X main.date={{ .CommitDate }} -X github.com/cosmos/cosmos-sdk/version.Name=archway -X github.com/cosmos/cosmos-sdk/version.AppName=archway -X github.com/cosmos/cosmos-sdk/version.Version={{ .Version }} -X github.com/cosmos/cosmos-sdk/version.Commit={{ .Commit }}
       - -linkmode=external
-      - -extldflags '-Wl,-z,muldefs -static -lm'
+      - -extldflags '-Wl,-z,muldefs --static -lm'
     tags:
       - netgo
       - muslc
+archives:
+  - id: w/version
+    builds:
+      - archwayd-linux-amd64
+      - archwayd-linux-arm64
+    name_template: "archwayd_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
+    wrap_in_directory: false
+    format: zip
+    rlcp: true
+    files:
+      - none*
+  - id: wo/version
+    builds:
+      - archwayd-linux-amd64
+      - archwayd-linux-arm64
+    name_template: "archwayd_{{ .Os }}_{{ .Arch }}"
+    wrap_in_directory: false
+    format: zip
+    rlcp: true
+    files:
+      - none*
+checksum:
+  # You can change the name of the checksums file.
+  # Default is `{{ .ProjectName }}_{{ .Version }}_checksums.txt`.
+  name_template: "{{ .ProjectName }}_{{ .Version }}_checksums.txt"
+dockers:
+  - use: buildx
+    goos: linux
+    goarch: amd64
+    image_templates:
+      - "ghcr.io/archway-network/{{ .ProjectName }}:{{ .Version }}-amd64"
+    build_flag_templates:
+      - "--pull"
+      - "--platform=linux/amd64"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
```

Comment:
Can we please also add the following tags to all images?

- `org.opencontainers.image.url=https://archway.io`
- `org.opencontainers.image.source=https://github.com/archway-network/archway`
- `org.opencontainers.image.licenses=Apache-2.0`